pkg/util/ciutil/systems/az_pipelines.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package systems
+
+import (
+	"fmt"
+	"os"
+)
+
+// AzurePipelinesCISystem represents the Azure Pipelines CI/CD system
+// that belongs to the Azure DevOps product suite.
+type AzurePipelinesCISystem struct {
+	BaseCISystem
+}
+
+var _ System = &AzurePipelinesCISystem{}
```

Comment:
We probably don't need this, since we convert the `AzurePipelinesCISystem` to the `System` interface elsewhere. Same comment in `generic.go`, and any other places where we do this. It's reasonable to do, as a way to state that the type does implement the given interface, but since we don't do it for every system we define, it's just noise.